tensorflow/python/ops/math_ops.py

```diff
@@ -99,9 +99,49 @@
 from tensorflow.python.util.tf_export import tf_export
 
 # Aliases for some automatically-generated names.
-linspace = gen_math_ops.lin_space
+#linspace = gen_math_ops.lin_space
 nextafter = gen_math_ops.next_after
 
+def linspace(start_in, stop_in, num, axis=0, name=None):
```

Comment:
The new argument has to be the last to avoid breaking backward compatibility

Reply:
I think this is done - let me know if there are issues.